extensions/2.0/Khronos/KHR_technique_webgl/README.md

```diff
@@ -14,122 +14,153 @@ Written against the glTF 2.0 spec.
 
 ## Overview
 
-TODO
+This extension allows glTF to define instances of shading techniques with external shader programs along with their parameterized values.  Shading techniques use JSON properties to describe data types and semantics for GLSL vertex and fragment shader programs.  
+
+This extension specification is targeting WebGL 1.0, and can be supported in any WebGL 1.0-based engine if the device supports the necessary WebGL extensions.
 
 ## glTF Schema Updates
 
+`KHR_techniques_webgl` introduces a list of the asset's shading techniques used in `techniques`. The technique and values are specified by adding a `KHR_techniques_webgl` extension to any glTF material.
+
+### Extending Materials
+
+If a `material` contains an `extensions` property and the `extensions` property defines its `KHR_techniques_webgl` property, then the specified shading technique can be used, as specified in the asset's `techniques`.
+
+```json
+"materials": [
+    {
+        "extensions": {
+            "KHR_techniques_webgl": {
+                "technique": 0,
+                "values" : {
+                    "u_light0Color": [
+                        1.0,
+                        0.8,
+                        1.0
+                    ]
+                }
+            }
+        }
+    }
+],
+```
+
+#### Technique
+
+The `technique` property points to the index of a technique listed in the asset's `KHR_techniques_webgl` `techniques` property.
+
+#### Values
+
+The `values` property defines the list of values, each of which is passed to the technique uniform of the same name, and when specified, overrides the corresponding uniform `value`.
```

Comment:
Since neither `.values` here nor `uniform.value` below are required, perhaps we need to say that at least one of the two must be present?